website/docs/r/dns_a_record.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Enables you to manage DNS A Records within Azure DNS.
 
+~> **Note:** [DNS has a throttle limit of 500 read (GET) operations per 5 minutes](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/request-limits-and-throttling#network-throttling). If you are provisioning a large scale of DNS records, please set a larger [read timeout](https://www.terraform.io/language/resources/syntax#operation-timeouts) then the default 5min. Alternatively, we’d recommend you split the resources out into smaller Terraform configurations to avoid the problem entirely.
```

Comment:
2 things here:

1. if we remove the locale this should redirect to the users language
2. rewording this slightly:

```suggestion
~> **Note:** [The Azure DNS API has a throttle limit of 500 read (GET) operations per 5 minutes](https://docs.microsoft.com/azure/azure-resource-manager/management/request-limits-and-throttling#network-throttling) - whilst the default read timeouts will work for most cases - in larger configurations you may need to set a larger [read timeout](https://www.terraform.io/language/resources/syntax#operation-timeouts) then the default 5min. Although, we'd generally recommend that you split the resources out into smaller Terraform configurations to avoid the problem entirely.
```